lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/view/contententry/detailattempttab/ContentEntryDeatilAttemptsSessionListScreen.kt

```diff
@@ -0,0 +1,169 @@
+                                if (attemptsSessionListItems.isSuccessful == true) "Success"
+                                else "Failure"
+                            }
+
+                            attemptsSessionListItems?.isCompleted == true -> "Completed"
+                            else -> "Incomplete"
+                        }, modifier = Modifier.padding(end = 8.dp))
+                },
+                headlineContent = {
+                    androidx.compose.material3.Text(
+                        text = when {
+                            attemptsSessionListItems?.isSuccessful != null -> {
+                                when (attemptsSessionListItems.isSuccessful) {
+                                    true -> "Passed"
+                                    false -> "Failed"
+                                    else -> ""
+                                }
+                            }
+
+                            else -> {
+                                if (attemptsSessionListItems?.isCompleted == true) {
+                                    "Completed"
+                                } else {
+                                    "Incomplete"
+                                }
+                            }
+                        }
+
+                    )
+                },
+                supportingContent = {
+                    Column {
+                        Row(
+                            modifier = Modifier.fillMaxWidth(),
+                            verticalAlignment = Alignment.CenterVertically
+                        ) {
+                            Icon(
+                                imageVector = Icons.Filled.Timer,
+                                contentDescription = "Icon",
+                                modifier = Modifier.padding(8.dp)
+                            )
+                            Text(
+                                text =
+                                        "$formattedDateAndTime"
+
+                            )
+
+                        }
+                        Row(
+                            modifier = Modifier.fillMaxWidth(),
+                            verticalAlignment = Alignment.CenterVertically
+                        ) {
+                            Icon(
+                                imageVector = Icons.Filled.Star,
+                                contentDescription = "Icon",
+                                modifier = Modifier.padding(8.dp)
+                            )
+                            Text(
+                                text = when {
+                                    attemptsSessionListItems?.maxScore != null -> {
+                                        "${((attemptsSessionListItems.maxScore ?: 0f) * 100).toInt()}% Score"
+                                    }
+
+                                    else -> {
+                                        "${(attemptsSessionListItems?.maxProgress)}% Completion"
```

Comment:
The app is used in multiple languages. All hardcoded strings must be replaced with using localized string resources.

Reply:
removed hardcoded strings